examples/problem_decomposition/ifci_mifno.ipynb

```diff
@@ -0,0 +1,554 @@
+    "- *energy_total*: Total energy of the fragment, with the correction term.\n",
+    "- *energy_correlation*: Total energy minus the mean-field energy.\n",
+    "- *epsilon*: Contribution to the MI summation.\n",
+    "- *correction*: MP2 correction, coming from the truncation of the virtual space in MI-FNO.\n",
+    "- *n_electrons*: Number of active electrons in the fragment.\n",
+    "- *n_spinorbitals*: Number of active spinorbitals in the fragment.\n",
+    "\n",
+    "Although not displayed here, the object also contains information about the frozen orbitals used for each of these fragments (\"frozen lists\")."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 2. Importing molecular coefficients files\n",
+    "\n",
+    "In order to build a `FermionOperator` or `QubitOperator` object compatible with our quantum algorithms, we can retrieve the MO coeffs exported by QEMIST Cloud. The \"result blob\" contains a problem handle for each fragment: the `retrieve_mo_coeff` method fetches them in the target directory provided by the user, and then loads the MO coeffs into the `MethodOfIncrementsHelper` object.\n",
+    "\n",
+    "- The default target folder is the folder where the user script is executed.\n",
+    "- If the user does not provide a path to a valid existing directory, the call returns an error."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "fno_fragments.retrieve_mo_coeff(data_folder)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 3. Reconstructing a fragment Hamiltonian\n",
+    "\n",
+    "We can use the MO coeffs to modify the molecular integrals, in order to take into account the FNO localization, using our \"frozen lists\".\n",
+    "\n",
+    "The `compute_fermionoperator` method handles the frozen orbitals for each fragment, and produces a `FermionOperator`. We are free to use the qubit mapping of our choice to produce a `QubitOperator` object, as input for a quantum algorithm."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Fragment (2,) mapped to 7 qubits.\n"
+     ]
+    }
+   ],
+   "source": [
+    "from tangelo import SecondQuantizedMolecule\n",
+    "from tangelo.toolboxes.operators import count_qubits\n",
+    "from tangelo.toolboxes.qubit_mappings import combinatorial\n",
+    "\n",
+    "# Needed to compute the molecular integrals.\n",
+    "mol = SecondQuantizedMolecule(xyz, q=charge, spin=spin, basis=basis)\n",
+    "\n",
+    "# Selection of a fragment and fetching the number of electrons and spinorbitals.\n",
+    "selected_fragment = \"(2,)\"\n",
+    "n_electrons, n_spinorbitals = fno_fragments.n_electrons_spinorbs(selected_fragment)\n",
```

Comment:
maybe this used to be complicated, but isnt the info in the dataframe ? is this function fishing the values from there? Is this function still useful ?

I have a feeling that a property or something should be returning individually electrons and spin orbitals. It does not matter for this PR, we can just acknowledge that if needed and move on.

Reply:
It wasn't, but now it is. It is the other way arounbd: the dataframe is fetching the data from the method by doing:
```python
df["n_electrons"], df["n_spinorbitals"] = zip(*df.index.map(self.n_electrons_spinorbs))
```